zeapp/gradle/libs.versions.toml

```diff
@@ -91,5 +81,5 @@ test-assertk = "com.willowtreeapps.assertk:assertk-jvm:0.27.0"
 test-junit = "junit:junit:4.13.2"
 test-compose-junit = "androidx.compose.ui:ui-test-junit4:1.5.3"
 test-compose-manifest = "androidx.compose.ui:ui-test-manifest:1.5.3"
-test-mockk = "io.mockk:mockk:1.13.8"
+test-mockk = "io.mockk:mockk:1.13.7"
```

Comment:
It was giving an error. https://github.com/mockk/mockk/issues/1173 They will fix in the next release